website/integrations/services/linkwarden/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Linkwarden
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Linkwarden
+
+> Linkwarden is an open-source collaborative bookmark manager to collect, organize and preserve webpages.
+>
+> -- https://linkwarden.app/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `linkwarden.company` is the FQDN of the Service install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## Service configuration
+
+To configure Linkwarden to use authentik, add the following values to your `.env` file:
```

Comment:
Is this only for Docker? Would one add these values to their `values.yaml` file for K8s?